core/jobs/io/firebase_io.py

```
@@ -0,0 +1,151 @@
+
+    These records are considered to be "weak" because they are NOT based on real
+    data. Instead, they are built using Oppia's internal association models,
+    under the assumption that they are consistent with "strong" (real) records.
+    """
+
+    def expand(
+        self, pbegin: pvalue.PBegin
+    ) -> beam.PCollection[firebase_adapters.WeakRecord]:
+        """Returns all of the "weak" records from Oppia's user & auth models."""
+
+        key_by_id = lambda model: (model.id, model)
```

Ruff (0.15.15)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `key_by_id` as a `def`

(E731)